File: CafeEase/CafeEase.Model/Order.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace CafeEase.Model
{
    public class Order
    {
        public int Id { get; set; }
        public DateTime OrderDate { get; set; }
        public decimal TotalAmount { get; set; }
        public string? Status { get; set; }
        public int UserId { get; set; }
        public int TableId { get; set; }
    }
}

File: CafeEase/UI/cafeease_desktop/lib/models/order.dart
import 'package:json_annotation/json_annotation.dart';

part 'order.g.dart';

@JsonSerializable()
class Order {
  int? id;
  DateTime? orderDate;
  double? totalAmount;
  String? status;
  int? userId;
  int? tableId;

  Order({
    this.id,
    this.orderDate,
    this.totalAmount,
    this.status,
    this.userId,
    this.tableId,
  });

  factory Order.fromJson(Map<String, dynamic> json) =>
      _$OrderFromJson(json);

  Map<String, dynamic> toJson() => _$OrderToJson(this);
}
